cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+DOCUMENTATION = '''
+---
+module: ovh_ip_loadbalancing_backend
+short_description: Manage OVH IP LoadBalancing backends
+description:
+    - Manage OVH (French European hosting provider) LoadBalancing IP backends
+version_added: "2.1"
+author: Pascal HERAUD @pascalheraud
+notes:
+    - Uses the python OVH Api U(https://github.com/ovh/python-ovh). \
+    You have to create an application (a key and secret) with a consummer \
+    key as described into U(https://eu.api.ovh.com/g934.first_step_with_api)
```

Comment:
Python multiline strings (started and stopped using triple quotes like ```'''````  and ```"""```` do not need the backslash as a line continuation character.  So you can remove all trailing backslashes at the end of lines within triple quoted strings.

I can't be certain but you may have added them in here because you weren't sure how yaml was parsing the documentation.  If so, the problem is not that the line needs to be continued.  Instead, it's that the indentation level is wrong.  In order for YAML to understand that all of this is one entry, you probably need to make sure the indent lines up with the other text, not with the character introducing the element.  IE:  The "Y" in You needs to be directly underneath the "U" in Uses.  Similarly, the "k" in key needs to be directly underneath those.